refinery/tool_manager/visualization_annotations/.gitignore

```diff
@@ -0,0 +1 @@
+*.json
```

Comment:
Can you explain this? (In person, if that's easier.) What's generating the files you want to ignore?